### PR TITLE
feat(addon-mobile): `SwipeActions` add new component

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
Closes #

### CODE REVIEW

**Review by splincode — APPROVED** [review]

**Comment by vladimirpotekhin** [comment]
wait https://github.com/taiga-family/taiga-ui/pull/6637 to fix examples

File: projects/addon-mobile/components/swipe-action/swipe-actions-auto-close.directive.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    },
+})
+export class TuiSwipeActionsAutoCloseDirective {
+    @Input()
+    autoClose = true;
+
+    constructor(@Inject(ElementRef) private readonly el: ElementRef<HTMLElement>) {}
+
+    onPointer(event: PointerEvent): void {
+        const target = tuiGetActualTarget(event);
```
Comment (waterplea) [inline projects/addon-mobile/components/swipe-action/swipe-actions-auto-close.directive.ts:18]: Maybe we do not need this const? Doesn't improve readability that much IMHO

File: projects/addon-mobile/components/swipe-action/swipe-actions-auto-close.directive.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    selector: 'tui-swipe-actions[autoClose]',
+    host: {
+        '(document:pointerdown.silent)': 'onPointer($event)',
+    },
+})
+export class TuiSwipeActionsAutoCloseDirective {
+    @Input()
+    autoClose = true;
+
+    constructor(@Inject(ElementRef) private readonly el: ElementRef<HTMLElement>) {}
```
Comment (waterplea) [inline projects/addon-mobile/components/swipe-action/swipe-actions-auto-close.directive.ts:15]: `inject`? We can then store `nativeElement` itself in `el` and make code shorter, maybe even get rid of `close` method?

File: projects/core/styles/mixins/mixins.less [hunk 3]
```diff
@@ -407,9 +407,7 @@
 
     &::-webkit-scrollbar,
     &::-webkit-scrollbar-thumb {
-        background: transparent;
-        width: 0;
-        height: 0;
+        display: none;
```
Comment (waterplea) [inline projects/core/styles/mixins/mixins.less:410]: Don't forget SCSS

**Review by waterplea — APPROVED** [review]

**Pull request merged**